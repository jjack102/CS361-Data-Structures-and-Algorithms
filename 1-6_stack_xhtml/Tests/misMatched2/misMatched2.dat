<html>
<head>
   <title>Traditional Greetings</title>
   <link  rel="stylesheet" type="text/css" href="test.css"/>
</head>
<body>
  <div class="greeting">
    <b>Hello</b><br/>world!
  </span>
  <br/>
</body>
</html>
