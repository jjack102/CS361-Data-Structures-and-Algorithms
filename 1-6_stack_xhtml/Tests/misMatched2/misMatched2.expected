Mismatched tags detected at tag </span>
